sections/reading_rust.tex

```diff
@@ -349,6 +362,14 @@ \section{Reading Rust}
   None => println!("No quotient."),
   Some(v) => println!("Quotient is {}.", v)
 }
+
+// ...or
+
+if let Some(q) = safe_div(num, denom) {
+  println!("Quotient is {}", q);
+} else {
+  println!("N", q);
```

Comment:
This looks wrong. `q` isn't available in the `else` branch, is it?

Reply:
Whoops, yep!